__tests__/app.test.js

```diff
@@ -27,3 +47,42 @@ describe("api/topics", () => {
     return request(app).get("/api/topic").expect(404);
   });
 });
+
+describe("api/articles/:article_id", () => {
+  test("GET:200 returns an article object with corresponding properties", () => {
+    return request(app)
+      .get("/api/articles/3")
+      .expect(200)
+      .then((response) => {
+        const article = response.body.article;
+        article.forEach((article) => {
+          expect(article.title).toBe("Eight pug gifs that remind me of mitch");
+          expect(article.topic).toBe("mitch");
+          expect(article.author).toBe("icellusedkars");
+          expect(article.body).toBe("some gifs");
+          expect(article.hasOwnProperty("created_at")).toBe(true);
+          expect(article.article_img_url).toBe(
+            "https://images.pexels.com/photos/158651/news-newsletter-newspaper-information-158651.jpeg?w=700&h=700"
+          );
+          expect(article.votes).toBe(0);
+          expect(article.article_id).toBe(3);
```

Comment:
Really good test, you've been as specific as you can with each property value which is great! 🏆 
Something you could use here is `toMatchObject` which could consolidate all your `expect` assertions into one single assertion